src/multitfa/analysis/variability.py

```diff
@@ -2,6 +2,13 @@
 
 import numpy as np
 from pandas import DataFrame, Series
+import os
+
+cwd = os.getcwd()
+
+logs_dir = cwd + os.sep + "logs"
+if not os.path.exists(logs_dir):
+    os.makedirs(logs_dir)
 
 
 def variability(model_variability, variable_list=None):
```

Comment:
I have been using this branch with CPLEX for a couple of days, including the variability, and it's been working great so far.